specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -734,9 +734,7 @@
         "sentiment",
         "confidenceScores",
         "offset",
-        "length",
-        "aspects",
-        "opinions"
+        "length"
```

Comment:
Shouldn't these be optional then?

Reply:
There is no "optional" tag as far as I know; they will be optional properties since those are removed from the "required" and those are NOT removed from the real "properties". :)